source/applications/advanced/color_balance.py

```diff
@@ -201,6 +201,8 @@ def _color_balance_calibration(camera: zivid.Camera, settings_2d: zivid.Settings
     return (corrected_red_balance, corrected_green_balance, corrected_blue_balance)
 
 
+
+
```

Comment:
This should be removed from all files.

Reply:
Simply removing these lines will cause the direct links to fail.. as they would be off by 2.